docs/technical-design/resovler-design.md

```diff
@@ -0,0 +1,146 @@
+# Resolver design
+
+## Background
+Resolver functions are responsible for handling requests made to the MC-Review GraphQL API and populating data for the response. Between receiving the request data and returning a response, the resolvers perform various tasks for generating the response.
```

Comment:
```suggestion
Resolver functions are responsible for handling requests made to the MC-Review GraphQL API and populating data for the response on the backend. Between receiving the request data and returning a response, the `app-api` resolvers perform various tasks for generating the response.
```

To me its always good to clarify backend or frontend when we dealing with something that crosses the boundary. Graphql as our api protocol has helpers/ data transformation/ utilities on both sides of the stack.